packages/core/src/behavior.ts

```diff
@@ -42,22 +42,21 @@ export type LifecycleSignal = typeof startSignal | typeof stopSignal;
  * @template TReceived The received event
  * @template TEmitted The emitted value
  */
-export interface Behavior<TReceived extends EventObject, TEmitted> {
+
+export interface Behavior<T extends EventObject, TEmitted = any> {
```

Comment:
to align with the rest of the codebase I think that we should use `TEvent` over `T` here

Reply:
Done